lektor/builder.py

```diff
@@ -11,7 +11,10 @@
 from itertools import chain
 
 import click
-from werkzeug.posixemulation import rename
+try:
+    from os import replace as rename
+except ImportError:
```

Comment:
Given that Lektor no longer supported python < 3.6, `os.replace` should always be available. (I think?)

Reply:
Oh, I didn't realize that, then I'll remove the fallback.

Reply:
Hi! I believe this will install Werkzeug 2.x instead of 0.16. Is there a way to do it on my system manually? I'm running Ubuntu 20.04 by the way.
Thanks!